docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+2021-02-19T11:45:33.239+0000 logsize=2749; maxlog=5000000
+2021-02-19T11:45:33.244+0000 Created new dir /var/opt/nuodb/archive/nuodb/demo
+2021-02-19T11:45:37.059+0000 archiveId=2; DB=demo; hostname=sm-database-nuodb-cluster0-demo-hotcopy-0
+2021-02-19T11:45:38.444+0000 path=/var/opt/nuodb/archive/nuodb/demo; atoms=0; catalogs=0
+2021-02-19T11:45:43.686+0000 Latest restore for cluster0 resolved to 20210219T114404
+2021-02-19T11:45:43.692+0000 Automatic archive repair will be performed for archiveId=2, source=20210219T114404, type=backupset, strip=1
+2021-02-19T11:45:43.749+0000 Restoring 20210219T114404; existing archive directores: total 12
+drwxr-xr-x  2 nuodb root 4096 Feb 19 11:45 demo
+drwxr-xr-x 33 nuodb root 4096 Feb 19 11:45 demo_moved
+drwxr-xr-x  2 nuodb root 4096 Feb 19 10:32 demo-save-20210219T103238
+2021-02-19T11:45:43.778+0000 (restore) recreated /var/opt/nuodb/archive/nuodb/demo; atoms=0
+2021-02-19T11:45:43.783+0000 Calling nuodocker to restore 20210219T114404 into /var/opt/nuodb/archive/nuodb/demo
+2021-02-19T11:45:45.470+0000 Finished restoring /var/opt/nuodb/backup/20210219T114404 to /var/opt/nuodb/archive/nuodb/demo. Created archive with archive ID 4
+2021-02-19T11:45:46.853+0000 Purging archive metadata archiveId=2
+...
+```
+
+### Distributed database restore with storage groups
+
+Database restore using user-defined storage groups is a special case of a database restore operation.
+The process is documented in the [Distributed database in-place restore](#distributed-database-in-place-restore) section. Several considerations need to be taken into account:
+
+- a complete set of archives serving all storage groups must be restored when performing database in-place restore
+- to ensure that backup coverage is complete, each storage group must be served by at least one HC SM
+
+A complete set of archives can be selected using several of the methods described in [Fine-grained archive selection](#fine-grained-archive-selection) section. NuoDB won't perform any special checks during database restore to ensure that the archives selected for a restore are a complete set of archives.
+If some of the storage groups are missing from the selection, their state won't be restored.
+
+> **NOTE**: For more information about storage groups, check [Using Table Partitions and Storage Groups](https://doc.nuodb.com/nuodb/latest/database-administration/using-table-partitions-and-storage-groups/)
+
+### Assisted database restore
+
+_Assisted database restore_ is a special case of database restore operation and allows complex restore operations to be executed easier in Kubernetes deployments.
+The _assisted database restore_ operation is initiated by installing the _restore_ chart with `restore.manual="true"` which creates a manual restore request.
+Once the database is restarted all database processes will block waiting for the user to perform the selected archives restore.
```

Comment:
I wouldn't say "Once the database is restarted" because none of the database processes are actually started. I would make clear what the point is of this mode by including a sentence like

> This mode blocks all database pods before allowing them to form a database, in order to give the user access to the persistent volumes used by an SM.